.github/workflows/ci-docs.yml

```diff
@@ -0,0 +1,19 @@
+name: ci
+
+on:
+  pull_request:
+    paths: # This expression needs to match the paths ignored on ci.yml.
```

Comment:
Should we be excluding any PR that has files that don't match the below?

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore says

> If at least one path matches a pattern in the paths filter, the workflow runs.

So if we had a PR with a `.go` file as well as a `.md` file, then this would run, right?

Maybe we should use `paths-ignore` with negated patterns?

Reply:
that's true, I will change to `paths-ignore`. Thanks for catching this.

Reply:
I will open a PR with changes in a few places after we merge this one so we can ensure things work as we expect them to work.